src/webp.imageio/webpoutput.cpp

```diff
@@ -124,6 +124,9 @@ WebpOutput::open(const std::string& name, const ImageSpec& spec, OpenMode mode)
         m_webp_config.quality = 100;
     }
 
+    // Lossless encoding (0=lossy(default), 1=lossless).
+    m_webp_config.lossless = m_spec.get_int_attribute("lossless", 0);
```

Comment:
I would recommend

    m_webp_config.lossless = (m_spec.get_string_attribute("compression") == "lossless");

that's a little more in keeping with how the other format writers allow compression methods to be specified.

Reply:
Done, thanks for the suggestion